chat/data-structures.md

```diff
@@ -0,0 +1,26 @@
+# Data Structures
+
+In this document we define data structures and definitions used in the chat api
+
+## Invite
+
+Invite is an encrypted payload used in wc_chatInvite which will be tracked by the json-rpc id to index on the client-side with the following params.
+
+```jsonc
+{
+  "message": string,
+  "account": string,
+  "signature": string, // optional
+}
+```
+
```

Comment:
I am suggesting to use this data structure as it is inconvenient to keep pubkey and invite separately 
we can think of `pubkey` as of envelope address and of `invite` as an envelope content.
that structure could be a param in api's `invite()` method and emitted in `chat_invite` event.
We could also think of it as of opened envelope and of:
```jsonc
{
   pubKey: string,
   invite: string
}
```
as closed envelope 

```suggestion

### Invite Envelope
```jsonc
{
   pubKey: string,
   invite: {
     "message": string,
     "account": string,
      "signature": string, // optional
   }
}

```